tinylog-impl/src/main/java/org/tinylog/writers/JsonWriter.java

```diff
@@ -170,22 +185,19 @@ private void addJsonObject(final LogEntry logEntry, final StringBuilder builder)
 			escapeCharacter("\n", "\\n", builder, start);
 			escapeCharacter("\r", "\\r", builder, start);
 
-			builder.append("\" ");
+			builder.append("\"");
 
 			if (i + 1 < jsonProperties.size()) {
 				builder.append(",").append(NEW_LINE);
 			}
 		}
-		builder.append(NEW_LINE).append("\t},");
+		builder.append(NEW_LINE).append("\t}").append(NEW_LINE).append("]");
 	}
 
 	private void escapeCharacter(final String character, final String escapeWith, final StringBuilder stringBuilder,
 			final int startIndex) {
-		for (
-			int index = stringBuilder.indexOf(character, startIndex);
-			index != -1;
-			index = stringBuilder.indexOf(character, index + escapeWith.length())
-		) {
+		for (int index = stringBuilder.indexOf(character, startIndex); index != -1; index = stringBuilder
+				.indexOf(character, index + escapeWith.length())) {
```

Comment:
For me, the previous format of the the for loop is a bit better readable.

Reply:
fixed in https://github.com/tinylog-org/tinylog/pull/206/commits/b954bc8446282378959266e9434f5eb130e0d10c